cli/src/main/java/denominator/cli/Denominator.java

```diff
@@ -426,6 +427,44 @@ void overrideFromEnv(Map<String, String> env) {
       }
     }
 
+    static void setProxyFromEnv() {
+      String envHttpProxy = System.getenv("HTTP_PROXY");
```

Comment:
Yeah, this is one of those rare moments I suggest using a guava class.. put
it on the calendar!